pkgs/os-specific/linux/acpi-call/default.nix

```diff
@@ -29,5 +30,6 @@ stdenv.mkDerivation rec {
     inherit (src.meta) homepage;
```

Comment:
Please set the homepage explicitly.

Reply:
Come on, no need to change random stuff in the expression.